src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const textData = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const numbers = [...textData].map((elem) => +elem.innerHTML.replace(/,/gi, ''));
+
+const total = numbers.reduce(
+  (accumulator, current) => accumulator + current,
+  0
+);
+
+const average = total / numbers.length;
+
+function numberToString(number = 0) {
```

Comment:
Check out this method
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
You can format the number easier with it